news_items/005_fractions_and_input_changes/index.html

```diff
@@ -0,0 +1,81 @@
+
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS '21)
+  has added some big improvements to PrairieLearn.
+</p>
+
+<h5>Fraction input</h5>
+
+<p>
+  One of the most-requested features ever! You can now enter fractions
+  instead of numbers:
+</p>
+
+<img src="approx_pi.png" width="600px">
+
+<p>
+  You can even enter fractions with decimal points, like <tt>10.4/12.6</tt>
+</p>
+
+
+
+<h5>Clear handling of invalid answers</h5>
+
+<p>
+  If you submit an answer that PrairieLearn can't understand,
+  like <code>asf#$%J34jkl</code> when it's expecting a number, then it
+  will now show you clearly that something is wrong:
+</p>
+
+<img src="invalid_input.png" width="260px">
+
+<p>
+  Best of all, PrairieLearn doesn't count invalid answers as
```

Comment:
This seems to indicate PL counted it before? Maybe rewrite?

Reply:
Well, it doesn't technically mean that it counted them before 😄 -- I didn't want to get into the complexities here.